pkg/storaged/dialog.js

```diff
@@ -262,6 +262,17 @@
             });
         }
 
+        function update_fields(trigger) {
+            def.Fields.forEach(function (f) {
+                if (f.TextInput && f.update) {
+                    $dialog.find('[data-field="' + f.TextInput + '"]').val(f.update(get_field_values(), trigger));
```

Comment:
The `find` here is always successful (if the dialog template isn't buggy).  It doesn't search for the trigger, but for the field that will have its `update` function called, and we should always be able to find that.  If it does fail for some reason, it will return an empty set and the following `val` will do nothing.